getting-started/choosing-a-development-host/mac-osx.md

```diff
@@ -1,4 +1,69 @@
 # Mac OSX
 
-Under development
+You can run Rocket.Chat for development on  Mac OSX. The following instructions has been tested on  M1 Macs.
 
+1. Install Meteor 
+
+    Currently Meteor don’t support Apple M1 native binaries as the latest meteor release uses Node.js 14 which doesn’t have support for it until now.
+
+    You can use Rosetta and enter the below command.
+
+    `arch -x86_64 npm install -g meteor`
+
+    or select Terminal in the Applications folder, press CMD(⌘)+I and check the “Open using Rosetta” option then using this command.
+
+    `npm install -g meteor`
```

Comment:
This is installing the latest meteor, currently isn't taking into account what version rocketchat needs :(

Reply:
this is expected and correct.. the host will always have the latest Meteor version, but the correct Meteor version will be used on each Meteor project (more comments on this below)

Reply:
I wrote this few months ago. Now i guess they have native support https://blog.meteor.com/meteor-2-5-1-with-native-mac-m1-support-fb1c4b6990b5. I'll just try running this project from scratch and update these